src/librawrtc/certificate.c

```diff
@@ -1,24 +1,25 @@
+
+#include <string.h>
+#include <limits.h>
+#include <rawrtc.h>
+#include "certificate.h"
+#include "utils.h"
```

Comment:
Do these lines need to be at the top or is reordering just `openssl/x509.h` sufficient?